.rubocop.yml

```diff
@@ -42,3 +42,7 @@ Style/Next:
   Enabled: false
 Style/IfUnlessModifier:
   Enabled: false
+Naming/FileName:
+  Enabled: true
```

Comment:
This is [enabled by default](https://rubocop.readthedocs.io/en/latest/cops_naming/#namingfilename), so this line is unnecessary. Let's see if GH will let me suggest a deletion:

```suggestion
```